### PR TITLE
refactor: wallet manager ui

### PR DESCRIPTION
**Pull request by BaojunCZ**

### CODE REVIEW

File: packages/neuron-ui/src/components/Settings/InputWalletPasswordDialog.tsx [hunk 1]
```diff
@@ -94,40 +83,46 @@ const InputWalletPasswordDialog = ({
...
+                  })
+                }
+              >
+                {t('common.cancel')}
+              </Button>
+            </ButtonDiv>
+          </Card.Footer>
+        </>
+      ) : (
+        <div>Network not found</div>
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/Settings/InputWalletPasswordDialog.tsx]: `Wallet not found`

File: packages/neuron-ui/src/services/UILayer.ts [hunk 2]
```diff
@@ -140,13 +142,22 @@ export const walletsCall = instantiateMethodCall(wallets) as {
...
   importMnemonic: (params: { name: string; mnemonic: string; password: string }) => void
   update: (params: { id: string; name?: string; password: string; newPassword?: string }) => void
-  delete: (params: { id: string; password: string }) => void
   export: (id: string) => void
   getActive: () => void
   activate: (id: string) => void
   backup: (id: string) => void
 }
 
+export const deleteWallet = (id: string, password: string, handleResult: any) => {
```
Comment (Keith-CY) [inline packages/neuron-ui/src/services/UILayer.ts]: Why move `delete` to an independent channel?

**Comment by Keith-CY** [comment]
CI failed

File: packages/neuron-ui/src/services/UILayer.ts (hunk 2 above)
Comment (BaojunCZ) [reply packages/neuron-ui/src/services/UILayer.ts]: It should validate password before deleting wallet.If password is wrong,I should notify user in the dialog.It likes sync method.

Comment (Keith-CY) [reply packages/neuron-ui/src/services/UILayer.ts]: In this case, pass id and password to the neuron-wallet together, and keep the dialog open.

If the password is correct, delete the wallet and broadcast to neuron-ui. neuron-ui hdies the dialog.

If the password is incorrect, send error message to neuron-ui, and neuron-ui render it on the dialog.

File: packages/neuron-ui/src/components/Settings/InputWalletPasswordDialog.tsx [hunk 3]
```diff
@@ -94,40 +83,46 @@ const InputWalletPasswordDialog = ({
...
-              })
-            }
-          >
-            {t('common.cancel')}
-          </Button>
-        </ButtonDiv>
-      </Card.Footer>
+      {wallet ? (
+        <>
+          <Card.Header>{`Please Enter ${wallet.name} Password`}</Card.Header>
```
Comment (ashchan) [inline packages/neuron-ui/src/components/Settings/InputWalletPasswordDialog.tsx]: Apply i18n for UI text.

File: packages/neuron-ui/src/components/WalletEditor/index.tsx [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { RouteComponentProps } from 'react-router-dom'
 import { Card, Form, Button, Alert } from 'react-bootstrap'
 import { useTranslation } from 'react-i18next'
 
-import { Routes } from '../../utils/const'
+// import { Routes } from '../../utils/const'
```
Comment (ashchan) [inline packages/neuron-ui/src/components/WalletEditor/index.tsx]: Please do not keep such comments. Just delete unused code.

File: packages/neuron-ui/src/components/WalletEditor/index.tsx [hunk 5]
```diff
@@ -92,7 +92,8 @@ export default (props: React.PropsWithoutRef<ContentProps & RouteComponentProps<
           wallet={myWallet}
           dispatch={dispatch}
           checkType={CheckType.EditWallet}
-          handle={() => props.history.push(`${Routes.SettingsWallets}`)}
+          // handle={() => props.history.push(`${Routes.SettingsWallets}`)}
```
Comment (ashchan) [inline packages/neuron-ui/src/components/WalletEditor/index.tsx]: Delete this instead of comment out as well.

Comment (BaojunCZ) [reply packages/neuron-ui/src/components/WalletEditor/index.tsx]: fixed

File: packages/neuron-ui/src/services/UILayer.ts (hunk 2 above)
Comment (BaojunCZ) [reply packages/neuron-ui/src/services/UILayer.ts]: fixed

**Pull request merged**